scripts/release.py

```diff
@@ -470,13 +541,30 @@ def main():
     release = Release(rpco_tag, github_token=token)
 
     if not args.existing_release:
-        release.publish_release()
-        release.update_milestones()
-    if not release.pre_release:
+        if not args.do_not_publish_release:
+            release.publish_release()
+        if not args.do_not_update_milestones:
+            release.update_milestones()
+
+    if (not args.do_not_file_docs_issue and
+            not release.pre_release):
         docs_repo = Repo(url=args.docs_repo_url, cache_dir=args.cache_dir,
                          bare=False)
         request_doc_update(token, docs_repo, release)
 
+    # Guess next cycle version number
+    if not args.future_tag:
+        future_tag = str(release.tag.next_revision())
+    else:
+        future_tag = str(args.future_tag)
+
+    # Update the future_tag into repo
+    # Then git add, commit and push.
```

Comment:
These comments seem to relate to update_repo_with_new_ver_number, I think you should either remove them or add a docstring to the function.